pages/common/croc.md

```diff
@@ -0,0 +1,32 @@
+# croc
+
+> Send and recieve files easily and security over any network.
+> More information: <https://github.com/schollz/croc>.
+
+- Send a file or folder:
+
+`croc send {{path/to/file}}`
+
+- Send a file or folder with a specific passphrase:
+
+`croc send --code {{passphrase}} {{path/to/file}}`
+
+- Recieve a file or folder a receiving machine:
+
+`croc {{passphrase}}`
+
+- Send and connect over a custom relay:
+
+`croc --relay {{ip_to_relay}} send {{path/to/file}}`
+
+- Recieve and connect over a custom relay:
+
+`croc --relay {{ip_to_relay}} {{passphrase}}`
+
+- Host your own croc relay with default ports:
```

Comment:
Perhaps this will read better? Generally we try and avoid `you` etc.
```suggestion
- Host a croc relay on the default ports:
```

Reply:
These two parts should be fixed now as well. Thanks!